### PR TITLE
Copy our grub config in place

### PR DESCRIPTION
**Pull request by jimmykarily**
Fixes https://github.com/kairos-io/kairos/issues/622

### CODE REVIEW

File: tools-image/build-arm-image.sh [hunk 1]
```diff
@@ -328,6 +333,8 @@ if [ -n "$EFI" ] && [  -n "$efi_dir" ]; then
   cp -rfv $efi_dir/* $EFI
 fi
 
+partprobe
```
Comment (jimmykarily) [inline tools-image/build-arm-image.sh:336]: not sure if this is needed, I remember I got some error reading partitions at some point. It doesn't hurt though.

File: tools-image/build-arm-image.sh [hunk 2]
```diff
@@ -417,6 +424,9 @@ cp -rfv /defaults.yaml $WORKDIR/persistent/cloud-config/01_defaults.yaml
 
 grub2-editenv $WORKDIR/state/grub_oem_env set "default_menu_entry=Kairos"
 
+# We copy the file we saved earier to the STATE partition
+cp -rfv "${tmpgrubconfig}" $WORKDIR/state/grubmenu
```
Comment (mudler) [inline tools-image/build-arm-image.sh:428]: same is needed for the `prepare_arm_image.sh` script

Comment (mudler) [reply tools-image/build-arm-image.sh:428]: https://github.com/kairos-io/osbuilder/blob/a868346eacfd356f351da6248f3b4ab1dda4be2e/tools-image/prepare_arm_images.sh#L109

Comment (mudler) [reply tools-image/build-arm-image.sh:428]: maybe we can do a common script to prepare the state directory? at least its consistent between all the scripts here

**Comment by mudler** [comment]
just thinking out loud here: what if we have instead a cloud config instruction, somewhere, to run this copy on first-boot if its not present? I think it should not replace this solution, but sounds a good safeguard if something goes off and someone mistakenly removes the file

**Review by mauromorales — APPROVED** [review]
cool, I'll try and see if this helps with the font as Dimitris suggested

**Comment by mauromorales** [comment]
tested it out but unfortunately doesn't fix the font issue

**Pull request merged**